test/spoof/error_checks.go

```diff
@@ -51,3 +51,7 @@ func isConnectionRefused(err error) bool {
 func isConnectionReset(err error) bool {
 	return err != nil && strings.Contains(err.Error(), "connection reset by peer")
 }
+
+func isEOF(err error) bool {
+	return err != nil && strings.Contains(err.Error(), "EOF")
```

Comment:
can we use: https://golang.org/pkg/io/#pkg-variables EOF from here and just equality compare/

Reply:
Sounds good.

Reply:
Pushed. Actually had to use a little different approach because of this error that I was getting on `cmp.Equal()`: 
```
panic: cannot handle unexported field at {*errors.errorString}.s:
	"errors".errorString
```
But anyway, I think this approach is even better. I also removed the test because it probably doesn't make sense anymore. We're using a built-in function now.